tests/CMakeLists.txt

```diff
@@ -67,9 +67,11 @@ add_test_case(channel_cancels_pending_tasks)
 
 if (NOT WIN32)
     add_test_case(channel_message_passing)
-    add_test_case(test_default_with_ipv6_lookup)
-    add_test_case(test_resolver_ipv6_address_lookup)
-    add_test_case(test_default_with_multiple_lookups)
+    if (NOT APPLE)
```

Comment:
Probably worth a comment here to explain why not apple

Reply:
ok